src/main/java/org/jabref/logic/preferences/JabRefCliPreferences.java

```diff
@@ -1819,6 +1825,44 @@ public LastFilesOpenedPreferences getLastFilesOpenedPreferences() {
         return lastFilesOpenedPreferences;
     }
 
+    @Override
+    public GitPreferences getGitPreferences() {
+        if (gitPreferences != null) {
+            return gitPreferences;
+        }
+
+        boolean autoPushEnabled = getBoolean("gitAutoPushEnabled", false);
+        String storedMode = getString("gitAutoPushMode", AutoPushMode.ON_SAVE.name());
+        AutoPushMode autoPushMode = AutoPushMode.fromString(storedMode);
+
+        String gitHubUsername = getString("gitHubUsername", "");
+        String gitHubPasskey = getString("gitHubPasskey", "");
```

Comment:
Please read on at org.jabref.logic.preferences.JabRefCliPreferences#getProxyPreferences how JabRef stores passwords.

--> We use com.github.javakeyring.Keyring - and DO NOT STORE PASSWORDS IN PLAIN TEXT